ui/src/components/EntitySearch/FacetedEntitySearch.scss

```diff
@@ -46,18 +52,81 @@
     margin-bottom: $aleph-grid-size*2;
   }
 
-}
+  .SearchFieldSelect {
+    @include small {
+      display: none;
+    }
+  }
 
-@media screen and (max-width: $aleph-screen-sm-max-width) {
-  .hide {
-    display: none;
+  .DualPane__inner-container {
+    position: relative;
   }
 
-  .show {
-    display: block;
+  .bp3-drawer {
+    box-shadow: none;
+  }
+
+  .bp3-overlay {
+    width: $aleph-sidebar-width;
+  }
+
+  &__side-placeholder.SidePane {
+    height: auto !important;
+    position: relative;
+    transition: width 120ms linear;
+    min-height: 100vh;
+    width: $aleph-sidebar-width !important;
+
+    .collapsed & {
+      width: 0 !important;
+      padding: 0;
+    }
+  }
+
```

Comment:
is the !important required for some reason? I've never been a fan.

Reply:
Mostly laziness tbh